packages/synthetix-main/contracts/modules/core/AccountModule.sol

```diff
@@ -20,6 +20,14 @@ contract AccountModule is IAccountModule, OwnableMixin, AccountRBACMixin, Associ
     error OnlyTokenProxyAllowed(address origin);
     error InvalidRole();
 
+    modifier onlyAccountToken() {
+        if (msg.sender != address(getAccountTokenAddress())) {
+            revert OnlyTokenProxyAllowed(msg.sender);
+        }
+
+        _;
+    }
+
```

Comment:
Only moved this up from below (I like modifiers up)

Reply:
Rename to `OnlyAccountTokenProxy`?